### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Ensenkei**
Implemented function according to the task and checklist.md

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_AMOUNT = days * 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= 7) {
```
Comment (etojeDenys) [inline src/calculateRentalCost.js:11]: better to move 7 and 3 also to some constants

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_AMOUNT = days * 40;
```
Comment (etojeDenys) [inline src/calculateRentalCost.js:7]: it's not a constant

**Review by etojeDenys — APPROVED** [review]
good job

File: src/calculateRentalCost.js (hunk 1 above)
Comment (Ensenkei) [reply src/calculateRentalCost.js:11]: Shure, thank you!

File: src/calculateRentalCost.js (hunk 2 above)
Comment (Ensenkei) [reply src/calculateRentalCost.js:7]: I've tried to use "let" and this error in VS

Comment (Ensenkei) [reply src/calculateRentalCost.js:7]: Just watched video with Solution of this task and i get what you mean. Thank you!

**Pull request closed without merging**